spec/inquisition/outputter/xlsx/builder_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe Inquisition::Outputter::XLSX::Builder do
+
+  after { FileUtils.rm_rf(Inquisition::Configuration.instance.output_path) }
+
+  describe '.call' do
```

Comment:
It would be better to follow the next sequence:
```ruby
describe '.call' do
  let(:builder) { instance_double(described_class) }

  before do
    allow(builder).to receive(:call)
    allow(described_class).to receive(:new).and_return(builder)
    described_class.call([])
  end

  it { expect(builder).to have_received(:call) }
end
```